maker-frontend/src/components/ConnectedTakers.tsx

```diff
@@ -13,11 +13,6 @@ const ConnectedTakers = ({ takers }: Props) => {
     return (
         <VStack spacing={3}>
             <Heading size={"sm"} padding={2}>{"Connected takers: " + takers.length}</Heading>
-            <UnorderedList>
```

Comment:
thanks, I did not think about this being a nuisance in production when I added it for early debugging tool for the team... 😅